setup.cfg

```diff
@@ -22,6 +22,13 @@ inline-quotes = single
 accept-encodings = utf-8
 
 
+[wemake_python_styleguide]
```

Comment:
I feel like we don't need these values here.
Because we should use default values by default in a project itself.

There's no need to duplicate them.

Reply:
Please, don't forget to remove these values.

Reply:
@mcproger